<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUpdateRegulationOfGroup</class>
 <widget class="QDialog" name="DialogUpdateRegulationOfGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Päivitä kaavamääräyksiä joukolle kaavakohteita</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Valitse kaavamääräys:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBoxRegulationTitles">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxShowOnlyUsedRegulations">
             <property name="text">
              <string>Listaa vain käytössä olevat määräykset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Kaavamääräyksen otsikko:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="plainTextEditRegulationTitle"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Määräysteksti:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="plainTextEditRegulationText"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Kaavamääräyksen kuvaus:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPlainTextEdit" name="plainTextEditRegulationDescription"/>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBoxShowAreaRegulations">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Listaa aluevarausten kaavamääräykset</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxShowSuplementaryAreaRegulations">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Listaa täydentävien aluekohteiden kaavamääräykset</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkBoxShowLineRegulations">
             <property name="text">
              <string>Listaa viivamaisten kohteiden kaavamääräykset</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxShowPointRegulations">
             <property name="text">
              <string>Listaa pistemäisten kohteiden kaavamääräykset</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Päivitysasetukset</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxUpdateLandUseClassificationsForPolygonFeatures">
            <property name="text">
             <string>Päivitä aluevarausten käyttötarkoitukset</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonSelectPolygonFeatures">
            <property name="text">
             <string>Valitse päivitettävät aluekohteet</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxUpdateLandUseClassificationsForSupplementaryPolygonFeatures">
            <property name="text">
             <string>Päivitä täydentävien aluekohteiden käyttötarkoitukset</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButtonSelectSupplementaryPolygonFeatures">
            <property name="text">
             <string>Valitse päivitettävät täydentävät aluekohteet</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxUpdateLandUseClassificationsForLineFeatures">
            <property name="text">
             <string>Päivitä viivamaisten kohteiden käyttötarkoitukset</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButtonSelectLineFeatures">
            <property name="text">
             <string>Valitse päivitettävät viivamaiset kohteet</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBoxUpdateLandUseClassificationsForPointFeatures">
            <property name="text">
             <string>Päivitä pistemäisten kohteiden käyttötarkoitukset</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButtonSelectPointFeatures">
            <property name="text">
             <string>Valitse päivitettävät pistemäiset kohteet</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxRemoveOldRegulationsFromSpatialFeatures">
            <property name="text">
             <string>Poista kaavakohteilta vanhat kaavamääräykset</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxUpdateRegulationTextsEvenIfSpatialFeatureHasMultipleRegulations">
            <property name="text">
             <string>Päivitä käyttötarkoitus vaikka kohteella olisi useita kaavamääräyksiä</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Sulje</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUpdate">
         <property name="text">
          <string>Tee päivitykset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUpdateAndClose">
         <property name="text">
          <string>Tee päivitykset ja sulje</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
